### PR TITLE
wappalyzer: Update patterns and icons

### PR DESCRIPTION
**Pull request by kingthorin**
Updates as of:
https://github.com/enthec/webappanalyzer/commit/f5e4db9d01c5257b79a3026e27f5b77d16ae5930

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by thc202** [comment]
The changelog could also be updated.

**Comment by kingthorin** [comment]
Done and done. I'll update our actions workflow in another PR.

File: addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/wappalyzer.html [hunk 1]
```diff
@@ -38,11 +38,17 @@ <H2>External Links</H2>
 <tr>
 	<td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
 	<td><a href="https://www.wappalyzer.com/">https://www.wappalyzer.com/</a></td>
-	<td>The Wappalyzer Homepage</td></tr>
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/wappalyzer.html:41]: Fix that, push coming.

On another PR, can this be tweaked somehow?
image
`Wappalyzer - Technology Detection`
`Technology Detection - Wappalyzer`
Something like that? To make it filterable either way.

**Comment by kingthorin** [comment]
Tweaked `WappalyzerJobResultData` link.

**Comment by thc202** [comment]
Thank you again!

File: addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/wappalyzer.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (thc202) [reply addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/wappalyzer.html:41]: That uses the title of the page, so should be just a matter of updating it accordingly.

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/wappalyzer.html:41]: Thanks.

**Pull request merged**